src/components/Vqb.vue

```diff
@@ -70,7 +71,8 @@ export default class Vqb extends Vue {
     this.initialValue = { ...params }; // make a copy
     if (index !== undefined) {
       this.editedStepIndex = index;
-      this.selectStep({ index: index - 1 });
+      const prevIndex = index === 0 ? index : index - 1;
```

Comment:
Another way would to do:

```suggestion
      const prevIndex = Math.max(index - 1, 0);
```

Don't know if it's more readable or not.